app/controllers/comments_controller.rb

```diff
@@ -1,17 +1,35 @@
 class CommentsController < ApplicationController
-  load_and_authorize_resource
+  skip_before_action :verify_authenticity_token
+
+  def index
+    @post = Post.find(params[:post_id])
+    @comments = @post.comments.includes(:user)
+    respond_to do |format|
+      format.json { render json: @comments }
+    end
+  end
```

Comment:
- Great job 👏🏻 The API endpoint returns all comments for a post.  ✔️   

![image](https://github.com/anderCM/blog-app/assets/79058364/8239a223-e495-4212-8a1c-18be32a9c2ff)